app/service/material-center/UserComponents.ts

```
@@ -0,0 +1,266 @@
+        component: component.component,
+        version: component.version
+      };
+
+      const componentQueryList = await this.find(paramComponent);  // 异步查询组件
+      let packageName = '';
+      if (component.npm && component.npm.package) {
+        packageName = component.npm.package;
+      }
+
+      // 根据包名和版本去查询组件库是否存在
+      const paramComponentLibrary = {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)